decompiler/frontend/binaryninja/handlers/types.py

```diff
@@ -117,7 +133,8 @@ def lift_struct_member(self, member: StructureMember, parent_struct_name: str =
         else:
             # if member is an embedded struct/union, the name is already available
             member_type = self._lifter.lift(member.type, name=member.name)
-        return ComplexTypeMember(0, name=member.name, offset=member.offset, type=member_type)
+            # This is still wrong for Pointers...
```

Comment:
If it is still wrong, then a separate issue should be opened :sweat_smile:

Reply:
I debugged it again and everything is actually working as intended. I removed the misleading comment in [e402d24](https://github.com/fkie-cad/dewolf/pull/353/commits/e402d24a706373da71d4bf8fb5c2fdeb2585a9c5).